examples/advent2024/day1_part1.carbon

```diff
@@ -0,0 +1,142 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
```

Comment:
These files don't really have much in terms of documentation. Would it be worth at least adding a README.md explaining what you're doing here?

Reply:
Done.